SwiftWisdomTests/BitMaskTests.swift

```diff
@@ -34,9 +34,9 @@ class BitMaskTests: XCTestCase {
     }
     
     func testDataInitializer() {
-        let data = "0x1f".ip_dataFromHexadecimalString()!
-        let mask = UInt8(ip_data: data)
-        let hex = data.ip_hexInt!
+        let data = "1f".ip_dataFromHexadecimalString()!
+        let mask = UInt8(ip_data: data)!
```

Comment:
Lotta force unwrapping here, could we have a guard instead that fails with XCTAssertFail or something?  What are people's general thoughts on this in tests?